Sources/MountebankSwift/Models/Imposter/Imposter+writeStubsToDisk.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+extension Imposter {
+    public func writeStubsToDisk(
+        directoryName: StaticString = #file,
+        testName: String = #function,
+        line: UInt = #line
+    ) throws {
+        let imposters = """
+        import Foundation
+        import MountebankSwift
+
+        let stubsFor\(name.map(sanitizeName)?.capitalized ?? ""): [Stub] = \(stubs.recreatable)
+        """
+
+        let dirURL = URL(fileURLWithPath: "\(directoryName)", isDirectory: false)
```

Comment:
nit: Maybe split up this `writeStubsToDisk` function, so that it does less things.

Reply:
Updated!